src/components/utils/array.js

```diff
@@ -0,0 +1,2 @@
+export const cleanFalsy = list => Array.filter(list, n => n)
```

Comment:
I guess it will given that we are using `babel-runtime` but worth testing it if you haven't already.